### ISSUE TITLE
Feature request: add option to collect full history of tab view

### ISSUE THREAD

**Issue by arcimboldo** [Stigmatoz/web-activity-time-tracker#25]
tl;dr provide the option to collect and export full history of open tabs: `start_date,end_date,page,duration`

Although the aggregate information provide incredibly useful data to know how much time we spend on a single page they do not provide any insight on *how* we spend this time. Chatting for 1h is very different than chatting 60 times for 1m. Collecting full historical data on the pages opened and when will allow an user to analyze the amount of interrupts (or distractions) during the day, and will allow to do further analysis on the correlation of different websites (every time I open facebook I then open amazon)

Rough estimate of the data used by this: one record ~256bytes, day ~12h*3600 seconds, 10s per tab => 360KiB per day => ~10MiB per month => ~120MiB per year (uncompressed). Some automatic pruning of the data might be needed.

**Comment by Stigmatoz** [comment]
A new version with a new codebase was released yesterday. And I plan to add a dashboard with some detailed information about web usage.
